_data/events.yml

```diff
@@ -5,6 +5,38 @@
 #   url: external url
 #   image: external url
 #   description: sentence or two (max 60 words)
+- name: dbt Learn (UTC-7/West Coast)
+  type: Education
+  start: July 15-16, 2020
```

Comment:
Our site doesn't accept date ranges, only start date. So you'll need to use "July 15, 2020"

I believe that's what's causing your build to break.

Reply:
I tried that, still not working. :(